pennylane/devices/default_clifford.py

```diff
@@ -103,28 +104,49 @@
     "StatePrep": None,
     "Snapshot": None,
     "Barrier": None,
+    "PauliError": "CORRELATED_ERROR",
+    "BitFlip": "X_ERROR",
+    "PhaseFlip": "Z_ERROR",
+    "DepolarizingChannel": "DEPOLARIZE1",
```

Comment:
Are these the only error channels stim supports?

Reply:
`Stim` support many more, but we don't have a corresponding (straightforward) `Channel` mapping for them in PennyLane.